src/types/chart/chart.ts

```diff
@@ -81,6 +81,10 @@ export interface TitleDesign {
   readonly color?: Color;
 }
 
+export interface ChartTitleConfiguration extends Omit<TitleDesign, "text"> {
+  readonly title: ChartTitle;
```

Comment:
Do we really need this object type ? Can't we simply use ChartTitle everywhere ? For me it's strange to have this nested structure that only contains one field

Reply:
Yes, I believe we do. I discussed this with ADRM in this comment: https://github.com/odoo/o-spreadsheet/pull/4024#issuecomment-2074837702

Reply:
That was not what I meant, but anyway, my question was wrong as I didn't figure we were extending the TitleDesign (I thoug you were creating a new object TitleConfiguration which only contains the ChartTitle, but it's not the case so you can ignore this comment :) )